app/client/cli/debug.go

```diff
@@ -50,6 +47,9 @@ var (
 		PromptTogglePacemakerMode,
 		PromptShowLatestBlockInStore,
 	}
+
+	defaultConfigPath  = getEnv("CONFIG_PATH", "build/config/config1.json")
```

Comment:
Hm, this certainly works because we set `CONFIG_PATH` for `client` pod. https://github.com/pokt-network/pocket/blob/dk-localnet-pocket-operator/build/localnet/cli-client.yaml#L24